### PR TITLE
OCPBUGS-7899: Bump sidecar timeouts for vsphere

### PR DESCRIPTION
**Pull request by gnufied**
Fixes https://issues.redhat.com/browse/OCPBUGS-7899

### CODE REVIEW

File: assets/controller.yaml [hunk 1]
```diff
@@ -118,6 +118,7 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --default-fstype=ext4
+            - --timeout=300s
```
Comment (jsafrane) [inline assets/controller.yaml:121]: I can see upstream has the same timeout in resizer and snapshotter, IMO it's worth bumping them too.

Comment (gnufied) [reply assets/controller.yaml:121]: agreed.

Comment (gnufied) [reply assets/controller.yaml:121]: fixed

**Pull request merged**